docs: debug run docs (#569)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/run.mdx

When creating a run, you may optionally provide a list of variable objects containing key and value attributes. These values apply to that run specifically and take precedence over variables with the same key applied to the workspace(e.g., variable sets). Refer to [Variable Precedence](/terraform/cloud-docs/workspaces/variables#precedence) for more information. All values must be expressed as an HCL literal in the same syntax you would use when writing Terraform code. Refer to [Types](/terraform/language/expressions/types#types) in the Terraform documentation for more details.

> Creating a run with `debugging_mode: true` will queue the run in debugging mode. This is equivalent to setting the `TF_LOG` environment variable to a value of `TRACE` for this run. See [Debugging Terraform](/terraform/internals/debugging) for more information.

Reviewer: ```suggestion
Setting `debugging_mode: true` enables debugging mode for the queued run only. This is equivalent to setting the `TF_LOG` environment variable to `TRACE` for this run. See [Debugging Terraform](/terraform/internals/debugging) for more information.
```

| `data.attributes.allow-config-generation`          | bool                 | `false`                                              | Specifies whether Terraform can [generate resource configuration](/terraform/language/import/generating-configuration) when planning to import new resources. When set to `false`, Terraform returns an error when `import` blocks do not have a corresponding `resource` block.                                                                                                                                                                                  |
| `data.attributes.auto-apply`                       | bool                 | Defaults to the [Auto Apply](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply) workspace setting.  | Determines if Terraform automatically applies the configuration on a successful `terraform plan`. |

> | `data.attributes.debugging-mode`                   | bool                 | `false`                                              | When set to `true`, the plan is queued with a verbose logging level. |

Reviewer: ```suggestion
| `data.attributes.debugging-mode`                   | bool                 | `false`                                              | When set to `true`, enables verbose logging for the queued plan. |
```
